mlir/include/mlir/Conversion/RockPasses.h

```diff
@@ -1,27 +1,27 @@
-//===- MIOpenPasses.h - Conversion Pass Construction and Registration -----===//
+//===- RockPasses.h - Conversion Pass Construction and Registration -----===//
 //
 // Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef MIOPEN_MLIR_CONVERSION_PASSES_H
-#define MIOPEN_MLIR_CONVERSION_PASSES_H
+#ifndef ROCK_MLIR_CONVERSION_PASSES_H
```

Comment:
This isn't a rock-specific file, so let's make this `rocMLIRPasses.h` and the include guard `MLIR_CONVERSION_ROCMLIRPASSES_H`